src/entitysdk/downloaders/emodel.py

```diff
@@ -0,0 +1,22 @@
+"""Download functions for EModel entities."""
+
+from pathlib import Path
+
+
+def download_hoc(client, emodel, hoc_dir="./hoc"):
```

Comment:
Please add type hints as it helps catching mistakes when lifting.

In addition, we shouldn't have default paths in general.

Reply:
done in latest commit